### ISSUE TITLE
Include sinh/asinh color transfer functions as options in render widget

### ISSUE THREAD

**Issue by jehibbard** [CARTAvis/carta-frontend#1992]
This request came during a CARTA v3.0 webinar given by J. Ott at NRAO. The user said these are available in DS9, so would be good to have in CARTA.

**Comment by d3v-null** [comment]
This would be a great addition, as there is currently no colour transfer function that is symmetric about 0 (other than linear of course).

**Comment by kswang1029** [comment]
@d3v-null you can try to apply a contrast to the linear transfer function to make a few symmetric effective transfer functions.
![Screen Shot 2022-09-06 at 10 48 54](https://user-images.githubusercontent.com/20819712/188536860-e642bd0a-261b-4db0-b524-c2385a62d513.png)

**Comment by d3v-null** [comment]
thanks @kswang1029 , but unfortunately at the moment the contrast doesn't sync with other images using raster matching. They do sync once when raster matching is first toggled but don't update.

**Comment by kswang1029** [comment]
> thanks @kswang1029 , but unfortunately at the moment the contrast doesn't sync with other images using raster matching. They do sync once when raster matching is first toggled but don't update.

yes, a to-do in the near future: https://github.com/CARTAvis/carta-frontend/issues/1959

**Comment by veggiesaurus** [comment]
@veggiesaurus to look into how things behave when changing contrast/bias
